### ISSUE TITLE
Issues with creating my first agent

### ISSUE THREAD

**Issue by jModeler** [minerllabs/minerl#698]
Hello! 
I'm following the [instructions here](https://minerl.readthedocs.io/en/latest/tutorials/first_agent.html) to create my first agent. I'm running into an error:

## A detailed walkthrough of the error, its code path and all known details is as follows:
```
-- Head --
Thread: Render thread
Stacktrace:
	at org.lwjgl.glfw.GLFW.glfwCreateWindow(GLFW.java:1842)
	at net.minecraft.client.MainWindow.<init>(MainWindow.java:91)
	at net.minecraft.client.renderer.VirtualScreen.create(VirtualScreen.java:21)
	at net.minecraft.client.Minecraft.<init>(Minecraft.java:408)

-- Initialization --
Details:
Stacktrace:
	at net.minecraft.client.main.Main.main(Main.java:154)

-- System Details --
Details:
	Minecraft Version: 1.16.5
	Minecraft Version ID: 1.16.5
	Operating System: Mac OS X (x86_64) version 10.16
	Java Version: 1.8.0_292, AdoptOpenJDK
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), AdoptOpenJDK
	Memory: 956382688 bytes (912 MB) / 1220542464 bytes (1164 MB) up to 3817865216 bytes (3641 MB)
	CPUs: 12
	JVM Flags: 1 total; -Xmx4G
	Launched Version: ~~NULL~~
	Backend library: LWJGL version 3.2.1 SNAPSHOT
	Backend API: NO CONTEXT
	GL Caps:
	Using VBOs: Yes
	Is Modded: Very likely; Jar signature invalidated
	Type: Client (map_client.txt)
	CPU: <unknown>
#@!@# Game crashed! Crash report saved to: #@!@# /Users/<username>/.pyenv/versions/3.11.2/lib/python3.11/site-packages/minerl/MCP-Reborn/./crash-reports/crash-2023-03-22_13.14.43-client.txt
```

I've looked through all other issues on this repo and haven't been able to find a solution that fixed this. Here's a list of things I've tried:

1. Putting all the code in [this page](https://minerl.readthedocs.io/en/latest/tutorials/first_agent.html) into a script and using `xvfb-run python3 <script_name>` instead (I get a similar error)
2. Checked if this was an [OpenGL issue](https://github.com/minerllabs/minerl/issues/618),  I seem to have OpenGL on my machine:

```
$ glxinfo | grep "OpenGL version"
OpenGL version string: 2.1 ATI-4.9.51
```
3. ~I haven't tried [this tip](https://minerl.readthedocs.io/en/latest/notes/performance-tips.html#faster-alternative-to-xvfb) yet and will check if this works in my case.~ Not sure if [this tip](https://minerl.readthedocs.io/en/latest/notes/performance-tips.html#faster-alternative-to-xvfb) applies to my machine since I have an AMD GPU. 

Please let me know if there's a simpler fix to get the first agent running on my machine. Thanks!

**Comment by Miffyli** [comment]
I see you are on Mac. Have you tried [these steps](https://github.com/minerllabs/minerl/issues/659#issuecomment-1306635414)?

If this works, I feel we should add this to the official documentation to help people out 😅

**Comment by jModeler** [comment]
Hello @Miffyli!
Thanks so much for sharing the steps. This works now on my machine.

I've documented the steps I've used:

- [For installing the package](https://jmodeler.github.io/post/resolving-install-errors-with-minerl/)
- [For running the first agent with minerl](https://jmodeler.github.io/post/resolving-errors-when-creating-your-first-minerl-agent/). This post is heavily based on the comment you linked. 

Feel free to link to these in the docs till you add more professionally written out steps 😁.  I'll close this issue -- thanks again!
